packages/server/src/controllers/settings/update.ts

```diff
@@ -84,6 +88,8 @@ export async function update(
     }
   }
 
+  const cacheKey = CACHE_KEYS.SITE_SETTINGS;
+
```

Comment:
Seems like redundant to define it with a variable here.

You can use the value directly.